cypress/integration/WebInterface/Test Cases/TestCasePopulationValues.spec.ts

```diff
@@ -604,7 +603,7 @@ describe('TC Pop value options are limited to those that are defined from Measur
         cy.get(EditMeasurePage.cqlEditorTab).click()
         cy.get(EditMeasurePage.cqlEditorTextBox).type('{enter}')
         cy.get(EditMeasurePage.cqlEditorSaveButton).click()
-        cy.wait(4500)
+        cy.wait(14500)
```

Comment:
is it possible to wait for an element vs an explicit wait?

Reply:
I don't like having to put that wait in either but it is for a wonlky issue around first creating the measure with CQL, on the back end, and, then, trying to add or adjust a group through the UI. We are having to put that whole block where we gotta log in, put some random change in the CQL, save that change and, then, wait for the change to take affect. I am seeing that it is taking longer and longer for that occur, hence the increase on the explicit wait. (edited) 
[2:09](https://semanticbits.slack.com/archives/G017E78C1FY/p1666033781665599)
If we don't do that whole step of logging in, after creating the measure with CQL and all via API call, and, then, immediately dive into adjusting a group or adding one to the measure, then we get invalid errors around the scoring not matching up with what is in the CQL.